get_nyt_articles.py

```diff
@@ -0,0 +1,88 @@
+'''
+Reads in articles from the New York Times API and saves
+them to a cache.
+'''
+
+import time
```

Comment:
Could you run pylint on this part, I think it can auto-format the the imports better 

http://google.github.io/styleguide/pyguide.html#21-lint